servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsClient.java

```diff
@@ -796,8 +796,8 @@ private void scheduleQuery0(final long remainingTtlNanos) {
             private void scheduleQuery0(final long remainingTtlNanos, final long originalTtlNanos) {
                 assertInEventloop();
 
-                final long delay = ThreadLocalRandom.current()
-                        .nextLong(remainingTtlNanos, addWithOverflowProtection(remainingTtlNanos, ttlJitterNanos));
+                final long delay = RandomUtils.nextLongInclusive(remainingTtlNanos,
+                                addWithOverflowProtection(remainingTtlNanos, ttlJitterNanos));
```

Comment:
looks like we now have that util method in more than one place. I know it's small, but is it still worth consolidating?

Reply:
I refactored it out of `RandomUtils`.